packages/golden_toolkit/lib/src/multi_screen_golden.dart

```diff
@@ -46,7 +51,8 @@ typedef DeviceSetup = Future<void> Function(Device device, WidgetTester tester);
 ///
 Future<void> multiScreenGolden(
   WidgetTester tester,
-  String goldenFileName, {
+  String name, {
+  DeviceFileNameFactory fileNameFactory = defaultDeviceFileNameFactory,
```

Comment:
I think we should remove the method level override and stick to just the global config